extensions/types/vscode-mssql.d.ts

```diff
@@ -489,112 +489,39 @@ declare module 'vscode-mssql' {
-		blockWhenDriftDetected: DacDeployOptionPropertyBoolean;
-		blockOnPossibleDataLoss: DacDeployOptionPropertyBoolean;
-		backupDatabaseBeforeChanges: DacDeployOptionPropertyBoolean;
-		allowIncompatiblePlatform: DacDeployOptionPropertyBoolean;
-		allowDropBlockingAssemblies: DacDeployOptionPropertyBoolean;
-		dropConstraintsNotInSource: DacDeployOptionPropertyBoolean;
-		dropDmlTriggersNotInSource: DacDeployOptionPropertyBoolean;
-		dropExtendedPropertiesNotInSource: DacDeployOptionPropertyBoolean;
-		dropIndexesNotInSource: DacDeployOptionPropertyBoolean;
-		ignoreFileAndLogFilePath: DacDeployOptionPropertyBoolean;
-		ignoreExtendedProperties: DacDeployOptionPropertyBoolean;
-		ignoreDmlTriggerState: DacDeployOptionPropertyBoolean;
-		ignoreDmlTriggerOrder: DacDeployOptionPropertyBoolean;
-		ignoreDefaultSchema: DacDeployOptionPropertyBoolean;
-		ignoreDdlTriggerState: DacDeployOptionPropertyBoolean;
-		ignoreDdlTriggerOrder: DacDeployOptionPropertyBoolean;
-		ignoreCryptographicProviderFilePath: DacDeployOptionPropertyBoolean;
-		verifyDeployment: DacDeployOptionPropertyBoolean;
-		ignoreComments: DacDeployOptionPropertyBoolean;
-		ignoreColumnCollation: DacDeployOptionPropertyBoolean;
-		ignoreAuthorizer: DacDeployOptionPropertyBoolean;
-		ignoreAnsiNulls: DacDeployOptionPropertyBoolean;
-		generateSmartDefaults: DacDeployOptionPropertyBoolean;
-		dropStatisticsNotInSource: DacDeployOptionPropertyBoolean;
-		dropRoleMembersNotInSource: DacDeployOptionPropertyBoolean;
-		dropPermissionsNotInSource: DacDeployOptionPropertyBoolean;
-		dropObjectsNotInSource: DacDeployOptionPropertyBoolean;
-		ignoreColumnOrder: DacDeployOptionPropertyBoolean;
 		doNotDropObjectTypes: DacDeployOptionPropertyObject;
 		excludeObjectTypes: DacDeployOptionPropertyObject;
-		ignoreTablePartitionOptions: DacDeployOptionPropertyBoolean;
-		doNotEvaluateSqlCmdVariables: DacDeployOptionPropertyBoolean;
-		disableParallelismForEnablingIndexes: DacDeployOptionPropertyBoolean;
-		disableIndexesForDataPhase: DacDeployOptionPropertyBoolean;
-		restoreSequenceCurrentValue: DacDeployOptionPropertyBoolean;
-		rebuildIndexesOfflineForDataPhase: DacDeployOptionPropertyBoolean;
-		isAlwaysEncryptedParameterizationEnabled: DacDeployOptionPropertyBoolean;
-		preserveIdentityLastValues: DacDeployOptionPropertyBoolean;
-		allowExternalLibraryPaths: DacDeployOptionPropertyBoolean;
-		allowExternalLanguagePaths: DacDeployOptionPropertyBoolean;
-		hashObjectNamesInLogs: DacDeployOptionPropertyBoolean;
-		doNotDropWorkloadClassifiers: DacDeployOptionPropertyBoolean;
-		ignoreWorkloadClassifiers: DacDeployOptionPropertyBoolean;
-		ignoreDatabaseWorkloadGroups: DacDeployOptionPropertyBoolean;
-		doNotDropDatabaseWorkloadGroups: DacDeployOptionPropertyBoolean;
+		booleanOptionsDictionary: { [key: string]: DacDeployOptionPropertyBoolean };
+	}
+
+	/*
+	* Interface containing option value and option name
+	*/
+	export interface IOptionWithValue {
```

Comment:
is this actually needed in the vscode-mssql extension? The DacFx options aren't exposed in UI anywhere there

Reply:
If it's not being used anywhere currently then just remove it